test/components/structures/AutocompleteInput-test.tsx

```diff
@@ -0,0 +1,244 @@
+/*
+Copyright 2022 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { screen, render, fireEvent, waitFor, within, act } from '@testing-library/react';
+
+import * as TestUtils from '../../test-utils';
+import AutocompleteProvider from '../../../src/autocomplete/AutocompleteProvider';
+import { ICompletion } from '../../../src/autocomplete/Autocompleter';
+import { AutocompleteInput } from "../../../src/components/structures/AutocompleteInput";
+
+describe('AutocompleteInput', () => {
+    const mockCompletion: ICompletion[] = [
+        { type: 'user', completion: 'user_1', completionId: '@user_1:host.local', range: { start: 1, end: 1 } },
+        { type: 'user', completion: 'user_2', completionId: '@user_2:host.local', range: { start: 1, end: 1 } },
+    ];
+
+    const constructMockProvider = (data: ICompletion[]) => ({
+        getCompletions: jest.fn().mockImplementation(async () => data),
+    }) as unknown as AutocompleteProvider;
+
+    beforeEach(async () => {
+        TestUtils.stubClient();
+    });
+
+    const getEditorInput = () => {
+        const input = screen.getByTestId('autocomplete-input');
```

Comment:
`getBy` already ensures that this element exists and explodes otherwise.
If you want to test, that some element exist you could do

```
expect(screen.queryByLabelText("something")).toBeInTheDocument();
```